src/software/ai/hl/stp/tactic/stop_tactic.h

```diff
@@ -1,7 +1,43 @@
 #pragma once
 
-#include "software/ai/hl/stp/action/move_action.h"
+#include "software/ai/hl/stp/action/move_action.h"  // TODO (#1888): remove this dependency
 #include "software/ai/hl/stp/tactic/tactic.h"
+#include "software/ai/intent/stop_intent.h"
+
+struct StopFSM
```

Comment:
similar comment here about this being exposed publicly, consider putting it in anonymous namespace.